### ISSUE TITLE
HelmCharts per OCI

### ISSUE THREAD

**Issue by wrenix** [ananace/personal-charts#4]
I like / prefer to pull charts as oci.
https://helm.sh/docs/topics/registries/

for mirroring like my images and so on.

**Comment by ananace** [comment]
This likely won't happen for a while, these Charts are used by some tooling which still requires the regular Helm repository layout.

**Comment by wrenix** [comment]
You could have both HelmRepository and OCI, like it:
- Prometheus: https://github.com/orgs/prometheus-community/packages?repo_name=helm-charts
- CollaboraOnline https://github.com/CollaboraOnline/online/pkgs/container/charts%2Fcollabora-online / https://github.com/CollaboraOnline/online

**Comment by ananace** [comment]
Yes, it's technically possible to run any number of simultaneous repositories for this, but I really don't want to push something that I can't even use myself.

I'm hoping to move some of these charts into new homes at some point soon, which should help open up some time for me to explore more modern tooling.

**Comment by 98jan** [comment]
any changes since then?
I am using harbor for mirroring all my helm charts where possible, but therefore I need an oci compliant helm chart.
Other option is to build a pipeline which syncs your helm charts to my registry.
